mne/epochs.py

```diff
@@ -1220,17 +1225,27 @@ def _get_data(self, out=True, verbose=None):
             self._getitem(good_idx, None, copy=False, drop_event_id=False,
                           select_data=False)
 
-        return data if out else None
+        if out:
+            if picks is None:
+                return data
+            else:
+                picks = _picks_to_idx(self.info, picks)
+                return data[:, picks]
+        else:
+            return None
+
 
-    def get_data(self):
+    @fill_doc
+    def get_data(self, picks=None):
         """Get all epochs as a 3D array.
 
         Returns
         -------
         data : array of shape (n_epochs, n_channels, n_times)
             A view on epochs data.
+        %(picks_all)s
```

Comment:
This is in the `Returns` section ...

Reply:
![giphy](https://user-images.githubusercontent.com/4321826/56657766-2b0fdc80-6699-11e9-8e53-1c9fb9ac632e.gif)

Reply:
And with that, `make docstrings` works for me locally.